example/lib/main.dart

```diff
@@ -127,15 +127,11 @@ class _AppHomeState extends State<AppHome> {
                     ),
                     child: Text('👈🏿 Swipe me Left OR Swipe me right 👉🏿 '),
                   ),
-                  iconOnLeftSwipe: Icons.arrow_forward,
                   leftSwipeWidget: FlutterLogo(
                     size: 30.0,
-                    colors: Colors.green,
                   ),
-                  iconOnRightSwipe: Icons.arrow_back,
                   rightSwipeWidget: FlutterLogo(
                     size: 30.0,
-                    colors: Colors.orange,
```

Comment:
The color property is removed on dev
https://github.com/flutter/flutter/blob/dev/packages/flutter/lib/src/material/flutter_logo.dart

Related PR: https://github.com/flutter/flutter/pull/62052